docs/UserGuide.md

```diff
@@ -208,7 +208,7 @@ Examples:
 
 ### Updating the amount a person owes you: `update`
 
-Updates the person at a given index in the list. The amount that the person owes the user is edited according to the filled in parameter.
+Updates the amount of a person at a given index in the list. The amount that the person owes the user is updated according to the amount parsed.
```

Comment:
I think should specify as "amount owed by a person" instead of "amount of a person". The latter sounds very vague.... and possibly misleading.